Tweak/Helper/KayokoHelper.xm

```diff
@@ -132,27 +140,133 @@ static void override_UIKeyboardLayoutStar_didMoveToWindow(UIKeyboardLayoutStar*
     CFNotificationCenterPostNotification(CFNotificationCenterGetDarwinNotifyCenter(), (CFStringRef)kNotificationKeyCoreHide, nil, nil, YES);
 }
 
+%group DictationAppearance
+
+%hook UIKeyboardDockItem
+
+- (id)initWithImageName:(NSString *)arg1 identifier:(id)arg2 {
+    if ([arg1 isEqualToString:@"mic"])
+        return %orig(@"doc.on.clipboard", arg2);
+    else if ([arg1 isEqualToString:@"mic.fill"])
+        return %orig(@"doc.on.clipboard.fill", arg2);
+    return %orig;
+}
+
+- (void)setImageName:(NSString *)arg1 {
+    if ([arg1 isEqualToString:@"mic"])
+        return %orig(@"doc.on.clipboard");
+    else if ([arg1 isEqualToString:@"mic.fill"])
+        return %orig(@"doc.on.clipboard.fill");
+    return %orig;
+}
+
+%end
+
+%hook UIKeyboardDockItemButton
+
+- (CGRect)imageRectForContentRect:(CGRect)arg1 {
+    CGRect origRect = %orig;
+    if (ABS(origRect.size.width - origRect.size.height) > 1.0) {
+        // adjust image to its 83% but keep its center
+        CGSize newSize = CGSizeMake(origRect.size.width * 0.833, origRect.size.height * 0.833);
+        CGPoint newOrigin = CGPointMake(origRect.origin.x + (origRect.size.width - newSize.width) / 2, origRect.origin.y + (origRect.size.height - newSize.height) / 2);
+        return CGRectMake(newOrigin.x, newOrigin.y, newSize.width, newSize.height);
+    }
+    return origRect;
+}
+
+%end
+
+%end
```

Comment:
I've made a small hack to change the icon of `dictation` to `pasteboard`. It looks better.